### PR TITLE
Add a script to pull translations from lokalise

### PR DESCRIPTION
**Pull request by tstirrat**
Adds `yarn i18n:pull` which will pull down all relevant translations.

There shouldn't be any en.json changes in subsequent runs, but this one syncs up lokalise with the local files...

Dependencies:

- [x] Fix up `export_para` plural issues (#648)

## How to test

Verify language updates look correct

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -80,34 +82,35 @@
...
+    "notification_title": "2-Week Exposure Profile",
+    "notification_today": "today",
+    "notification_warning_text": "If a Healthcare Authority exists in your area you can subscribe to obtain regular updates of exposure risks.",
+    "notifications_exposure_format": "%1$s days ago you crossed paths someone infectious for %1$s minutes.",
+    "notifications_exposure_format_today": "Today you crossed paths with someone infectious for %1$s minutes.",
+    "notifications_exposure_format_yesterday": "Yesterday you crossed paths with someone infectious for %1$s minutes.",
+    "notifications_no_exposure": "No known exposures to COVID-19 during the last two weeks.",
+    "overlap_found_button_label": "Public Data Loaded",
+    "overlap_no_results_button_label": "Public Data Loaded",
+    "overlap_para": "{\"one\":\"The green trail represents your location history\\n\\nThe light purple circles represent the public dataset\",\"other\":\"\"}",
```
Comment (tstirrat) [inline app/locales/en.json:101]: this is incorrect. needs fixing in lokalise

Comment (amasurkar-verimatrix) [reply app/locales/en.json:101]: do we need to fix this in both places or just on lokalise?

File: .lokalise.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+project-id: 331553825e9baabce4a297.57936175
+token: 2c6ba3a71fcf8cb305971cb7822bf8c510794d26 # read-only token
```
Comment (alpita-masurkar) [inline .lokalise.yml:2]: there's an id and token in this file here. what is this token for?
is it okay to leave it like this in an opensource project?

Comment (tstirrat) [reply .lokalise.yml:2]: It's read-only so it doesn't uncover anything that isn't already in the exported files.

Alternatively we could use an environment variable and you would need to create your own read only token on lokalise:

`LOKALISE_TOKEN=<my-token> yarn i18n:pull`

Actually, lets go with that.

File: app/locales/en.json (hunk 1 above)
Comment (tstirrat) [reply app/locales/en.json:101]: We need to fix in both places, I will make a PR now

File: app/locales/en.json [hunk 3]
```diff
@@ -41,11 +41,12 @@
...
-    "import_step_1": "Adding location data from Google will give you a head start on building your recent locations.",
-    "import_step_2": "Before you can import, you must first 'Take out' your location data from Google.",
-    "import_step_3": "Visit Google Takeout and export your Location History using following settings: \n1. Delivery method: \"Add to Drive\" \n2. Frequency: \"Export once\" \n3. File type & size: \".zip\" and \"1GB\" \n4. Google sends an email when the export is ready \n5. Return here to import locations. Import options: \n- Import from Google Drive \n- Download from browser, then import from local phone files. Make sure to be on WiFi network as files can be big.",
+    "import_step": "{\"one\":\"Adding location data from Google will give you a head start on building your recent locations.\",\"other\":\"Before you can import, you must first 'Take out' your location data from Google.\"}",
     "import_takeout": "Visit Google Takeout",
     "import_title": "Import Locations",
+    "language_change_alert_cancel": "Cancel",
+    "language_change_alert_proced": "Restart",
+    "language_change_alert_title": "App need to restart in order to use this language",
```
[the remark this answers is not in the document]
Comment (tstirrat) [reply app/locales/en.json:49]: This should be fixed in lokalise

File: app/locales/en.json (hunk 1 above)
Comment (tstirrat) [reply app/locales/en.json:101]: Fixing in #648

**Comment by alpita-masurkar** [comment]
@tstirrat This is really good! Once the conflicts are fixed, is this ready to be merged?

**Comment by tstirrat** [comment]
> @tstirrat This is really good! Once the conflicts are fixed, is this ready to be merged?

Yep, I will update and sync sooon. We need #648 merged before this too.

**Comment by Patrick-Erichsen** [comment]
Language updates look correct as far as I can tell - could you give a little more context as to what Lokalise is doing in the pull script though?

**Pull request merged**